ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+import java.util.Random;
+
+public enum Color {
+    RED, BLUE, GREEN, YELLOW, ORANGE;
+
+    private static final Random random = new Random();
+
+    public static Color getRandomColor() {

```

Comment:
```suggestion
    private final Random random = new Random();

    public Color getRandomColor() {

```
checklist issue, don't use static methods in this task